### PR TITLE
Improve minor code changes

### PR DESCRIPTION
**Pull request by smic1909**
**Description**
Small code improvements

**Tested scenarios**
No additional tests needed since no logic has changed.

**Fixed issue**
This change is not related to any created Github Issue

### CODE REVIEW

**Comment by rikterbeek** [comment]
Hi @smic1909,

Many thanks for the Pull Request! A lot of code is generated by generator so makes perfect sense to clean this code. As it does have a lot of changes we need to be carefull and as well test all our e2e tests so this will take some time. When I manually review it I do see one major concern would be great if you can fix that.

Regards,
Rik
Adyen

File: src/main/java/com/adyen/model/checkout/InputDetail.java [hunk 1]
```diff
@@ -64,12 +64,7 @@ public InputDetail configuration(Map<String, String> configuration) {
     }
 
     public InputDetail putConfigurationItem(String key, String configurationItem) {
-
-        if (this.configuration == null) {
-            this.configuration = null;
```
Comment (martinsrenato) [inline src/main/java/com/adyen/model/checkout/InputDetail.java:69]: I think this line was wrong before and should be fixed, not removed.
If configuration is null, a new map should be created so it prevents nullPointerExceptions to happen on the next lines
`configuration = new HashMap<>();`

File: src/main/java/com/adyen/model/checkout/PaymentMethod.java [hunk 2]
```diff
@@ -87,12 +87,7 @@ public PaymentMethod configuration(Map<String, String> configuration) {
     }
 
     public PaymentMethod putConfigurationItem(String key, String configurationItem) {
-
-        if (this.configuration == null) {
-            this.configuration = null;
```
Comment (martinsrenato) [inline src/main/java/com/adyen/model/checkout/PaymentMethod.java:92]: I think this line was wrong before and should be fixed, not removed (same case as in InputDetail)

File: src/main/java/com/adyen/model/checkout/RecurringDetail.java [hunk 3]
```diff
@@ -95,12 +95,7 @@ public RecurringDetail configuration(Map<String, String> configuration) {
     }
 
     public RecurringDetail putConfigurationItem(String key, String configurationItem) {
-
-        if (this.configuration == null) {
-            this.configuration = null;
```
Comment (martinsrenato) [inline src/main/java/com/adyen/model/checkout/RecurringDetail.java:100]: Same as other putConfigurationItem methods

File: src/main/java/com/adyen/model/checkout/Redirect.java [hunk 4]
```diff
@@ -49,12 +49,7 @@ public Redirect data(Map<String, String> data) {
     }
 
     public Redirect putDataItem(String key, String dataItem) {
-
-        if (this.data == null) {
-            this.data = null;
```
Comment (martinsrenato) [inline src/main/java/com/adyen/model/checkout/Redirect.java:54]: Same as the putConfigurationItem case

File: src/main/java/com/adyen/model/checkoututility/OriginKeysResponse.java [hunk 5]
```diff
@@ -41,12 +41,7 @@ public OriginKeysResponse originKeys(Map<String, String> originKeys) {
     }
 
     public OriginKeysResponse putOriginKeysItem(String key, String originKeysItem) {
-
-        if (this.originKeys == null) {
-            this.originKeys = null;
```
Comment (martinsrenato) [inline src/main/java/com/adyen/model/checkoututility/OriginKeysResponse.java:46]: Same as the putConfigurationItem case

File: src/main/java/com/adyen/model/checkout/InputDetail.java (hunk 1 above)
Comment (smic1909) [reply src/main/java/com/adyen/model/checkout/InputDetail.java:69]: Fixed.

File: src/main/java/com/adyen/model/checkout/PaymentMethod.java (hunk 2 above)
Comment (smic1909) [reply src/main/java/com/adyen/model/checkout/PaymentMethod.java:92]: Fixed.

File: src/main/java/com/adyen/model/checkout/RecurringDetail.java (hunk 3 above)
Comment (smic1909) [reply src/main/java/com/adyen/model/checkout/RecurringDetail.java:100]: Fixed.

File: src/main/java/com/adyen/model/checkout/Redirect.java (hunk 4 above)
Comment (smic1909) [reply src/main/java/com/adyen/model/checkout/Redirect.java:54]: Fixed.

File: src/main/java/com/adyen/model/checkoututility/OriginKeysResponse.java (hunk 5 above)
Comment (smic1909) [reply src/main/java/com/adyen/model/checkoututility/OriginKeysResponse.java:46]: Fixed.

**Comment by smic1909** [comment]
> Hi @smic1909,
> 
> Thanks for fixing the issues reported on comments. The methods are functioning correctly now.
> It seems, however, that your changes added some wildcard imports ( **'*'** ) on some classes, and that causes a checkstyle failure on the [Travis CI build](https://travis-ci.org/github/Adyen/adyen-java-api-library/builds/670269637).
> Could you please revert that and add the specific imports needed?
> 
> Regards,
> Renato
> Adyen

Sure. Replaced the star import. Please validate.

**Pull request merged**